src/js/view/controls/controls.js

```diff
@@ -141,7 +141,9 @@ export default class Controls extends Events {
         }
 
         // Floating Close Button
-        const floatingConfig = model.get('floating');
+        let floatingConfig = model.get('floating');
+        floatingConfig = floatingConfig.disabled ? null : floatingConfig;
```

Comment:
`model.get('floating`)` is `undefined` when it is not set in config.
`floatingConfig.disabled` will throw exception in that case.